### PR TITLE
Adds markers

### PR DESCRIPTION
**Pull request by ckdesir**
Includes some more stuff I did with the Maps API before skipping week 4, adds markers to the map + adds a custom theme.

### CODE REVIEW

File: portfolio/src/main/webapp/script-home.js [hunk 1]
```diff
@@ -139,8 +144,145 @@ function deleteComments() {
  * Creates a map using the Google Maps API.
  */
 function initMap() {
-  const map = new google.maps.Map(document.getElementById('map'), {
+  // A custom color scheme of the map + basic properties for initialization
+  // (like initial center of map + zoom level)
+  const mapOptions = {
```
Comment (chukwuemekagooglecorp) [inline portfolio/src/main/webapp/script-home.js:149]: mapOptions and mapLabel seem to be non-changing data. Make them global constants

File: portfolio/src/main/webapp/script-home.js [hunk 2]
```diff
@@ -139,8 +144,145 @@ function deleteComments() {
...
+    },
+    {
+      name: 'Kansas City, MO',
+      location: new google.maps.LatLng(39.0997, -94.5786),
+      content: 'Another \"field trip\", the food and vibes here were ' +
+          'amazing'
+    },
+  ];
+  // Constructs the polygon outlining the Bermuda Triangle
+  const bermudaTriangle = new google.maps.Polygon({
```
Comment (chukwuemekagooglecorp) [inline portfolio/src/main/webapp/script-home.js:251]: same with bermuda triangle. in fact, if you want, you can create a js file called "map_constants.js" and dump these constants there. 

I forget whether javascript file names should have underscores or dashes

File: portfolio/src/main/webapp/map-constants.js [hunk 3]
```diff
@@ -0,0 +1,127 @@
+/** 
+ * A custom color scheme of the map + basic properties for initialization
+ * (like initial center of map + zoom level).
+ */
+const mapOptions = {
```
Comment (chukwuemekagooglecorp) [inline portfolio/src/main/webapp/map-constants.js:5]: If these are constants, they should be capital, right? So this should be MAP_OPTIONS

Comment (ckdesir) [reply portfolio/src/main/webapp/map-constants.js]: YES

**Pull request merged**